tools/ci/taskcluster-run.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import argparse
+import gzip
+import logging
+import os
+import shutil
+import subprocess
+
+browser_specific_args = {
+    "firefox": ["--install-browser", "--reftest-internal"]
+}
+
+def tests_affected(commit_range):
+    output = subprocess.check_output([
```

Comment:
You could, of course, just import and use the function directly rather than going via a process.

Reply:
@jgraham wouldn't that require hacking sys.path? If so, I'm in favour of forking.

Reply:
Yes. I think it makes more long term sense to move all of this into `wpt run` and not have another wrapper script at all. So I don't object to defering that change here.